inst/vign/taxize_case_study.md

```diff
@@ -0,0 +1,103 @@
+<!--
+%\VignetteEngine{knitr::knitr}
+%\VignetteIndexEntry{Strategies for programmatic name cleaning}
```

Comment:
seems the title here  should be `Case study - resolving Species Plantarum names`